### ISSUE TITLE
How would I create a VTOL mission and simulate in gazebo classic

### ISSUE THREAD

**Issue by ryancramuh** [mavlink/MAVSDK-Python#735]
**To preface: I am extremely new to MAVSDK, PX4, and QGroundControl.

I am trying to simulate a VTOL mission with where I start at takeoff and Fixed-Wing fly a short distance (about 1 mile) to the mission area, hover around a couple points and return to the takeoff area. I tried simulating but it seems the mission.py example is for quadcopter simulation, since PX4 shows "WARN  [mission_feasibility_checker] Mission rejected: Landing waypoint/pattern required"

How would one create a python script to run a VTOL mission like this? How different is it from the mission.py example script?

Thanks for any ideas, they would be very appreciated

**PS... I would love to contribute to increasing VTOL support when I get a better idea of how things work

**Comment by julianoes** [comment]
Right, there is currently no VTOL/fixedwing support using the Mission plugin. Instead you have to use MissionRaw and manually assemble the mission items required. It's a bit harder to do. One way to go about it is to create a mission that works in QGC, upload to PX4, then download again and now save/export the items and then import this file into MAVSDK, or manually look at the JSON and convert it to MAVSDK MissionRaw items.
